4-Ansible_Intro/set-ssh.yml

```diff
@@ -0,0 +1,97 @@
+- name: Set SSH environment 
+  hosts: moomin_hosts
+
+  tasks:
+    - name: Create ~/.ssh dir
+      file:
+          path: /home/moomin/.ssh
+          state: directory
+   
+    - name: Create `authorized_keys` if absent
+      file:
+          path: /home/moomin/.ssh/authorized_keys
+          state: touch
+
+- name: Set SSH keys to host-1
+  hosts: host-1
+  
+  tasks:
+    - name: Copy ssh private key to host-1
+      copy:
+          src: host-1_rsa
+          dest: /home/moomin/.ssh/id_rsa
+          owner: moomin
+          group: moomin
+          mode: 0600
+
+- name: Copy SSH key to host-2
+  hosts: host-2
+  
+  tasks:
+    - name: Copy ssh private key to host-2
+      copy:
+          src: host-2_rsa
+          dest: /home/moomin/.ssh/id_rsa
+          owner: moomin
+          group: moomin
+          mode: 0600
```

Comment:
idk, the task was to "setup SSH keys"

After adding this keys to remote hosts it's now available to login from one host to another passwordless, since we didn't set the password while creating new user.